### PR TITLE
Converted status field to Submit/Save buttons

### PR DESCRIPTION
**Pull request by ashley**
For Issue #166 
* Removed status field from add and edit view. They would only appear if the status field was not found, which also means the view will use the original submit/save buttons.
* List view should still include status field; they should be updated as well
* Editing Draft events: Options to announce, and save changes will appear
* Editing Announced events: Options to cancel the event will appear
* Editing Canceled events: Options to un-cancel and delete events will appear
* Did basic QA testing of adding/editing events with different status.

Small bug: When updating add and edited resources, users have to refresh the list view to see the updates.

### CODE REVIEW

**Review by ashleytqy — COMMENTED** [review]
this looks great!! 💯

a few design concerns –
when we are editing a `draft` event, having `save changes`, `announce event` and `delete event` as call to actions may be confusing. also, if we want to change the time & announce the event, we have to change the time, click `save changes`, go back to the edit page, before we can `announce event`.

a possible solution: on the edit page – instead of  `save changes`, we can do `save changes and keep as draft`, `save changes and announce event`, and `delete event`.

Or, on the list page, have `announce event` / `uncancel event` / `cancel` event (rather than have it on the edit page, which only has the `save changes` button)
screen shot 2018-04-09 at 8 09 18 pm

File: app/partials/actions/event/event-add.html [hunk 1]
```diff
@@ -18,7 +18,11 @@ <h5>{{rdesc.attributes.description}}</h5>
 		ng-include="'/partials/fields/default-field.html'">
 	</div>
 
-	<div class="form-group">
-		<button class="btn btn-primary" ng-click="createResource(model, rdesc)">Create</button>
+	<div class="form-group" ng-if="statusFound">
+		<button class="btn btn-primary" ng-click="createResource(model, rdesc, 'announced')">Announce Event</button>
+		<button class="btn btn-info" ng-click="createResource(model, rdesc, 'draft')">Draft</button>
```
Comment (ashleytqy) [inline app/partials/actions/event/event-add.html]: Can we do `Save as Draft` instead of `Draft`?

**Comment by ashley** [comment]
- I renamed the buttons so that the original "Save Changes" button is now, "Save Changes as x event" where x is the current status.
- I also changed the announce event button to green so users can differentiate them more.

**Comment by ashleytqy** [comment]
@ashley screenshots would be helpful here !

**Comment by ashley** [comment]
**Event-Add Page**
screen shot 2018-04-10 at 7 30 59 pm

**Event-Edit Canceled Page**
screen shot 2018-04-10 at 7 30 06 pm

**Event-Edit Announced Page**
screen shot 2018-04-10 at 7 30 25 pm

**Event-Edit Draft Page**
screen shot 2018-04-10 at 7 30 41 pm

**Review by ashleytqy — APPROVED** [review]
💯

**Pull request merged**